UI/window-basic-main.cpp

```diff
@@ -1246,10 +1246,6 @@ void OBSBasic::Load(const char *file, bool remigrate)
 			}
 		}
 
-		config_set_string(App()->GetUserConfig(), "Basic",
-				  "SceneCollection", name.c_str());
-		config_set_string(App()->GetUserConfig(), "Basic",
-				  "SceneCollectionFile", name.c_str());
```

Comment:
These lines were originally added as part of https://github.com/obsproject/obs-studio/pull/11038 to fix a bug where scene collections could be erased (#11037). Can we confirm that that bug does not resurface from removing these lines?

Reply:
The values are always set by `ActivateCollection` now, right before `Load` itself is called:

https://github.com/obsproject/obs-studio/blob/06642fdee48477ab85f89ff670f105affe402df7/UI/window-basic-main-scene-collections.cpp#L758-L763

According to MSVC and Xcode that's the only function that calls `Load` so both config values should always be set with the added bonus that the collection name is not downgraded to a "safe" name for the file system.